src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -583,11 +579,7 @@ public function handleException(\Throwable $exception)
         }
 
         $exceptionHandler = $this->exceptionHandler;
-        if ((!\is_array($exceptionHandler) || !$exceptionHandler[0] instanceof self || 'sendPhpResponse' !== $exceptionHandler[1]) && !\in_array(\PHP_SAPI, ['cli', 'phpdbg'], true)) {
-            $this->exceptionHandler = [$this, 'sendPhpResponse'];
-        } else {
-            $this->exceptionHandler = null;
-        }
```

Comment:
This was preventing a blank error page when an error ocurrs on custom "exception normalizers", "exception controllers" or "exception listeners". Do you have a better idea to catch/show these errors?

Reply:
Restored.